### PR TITLE
Update rubygem-fog-vsphere to 3.6.0

### PR DESCRIPTION
**Pull request by chris1984**
@ekohl here is the packaging pr for the new fog-vsphere gem

### CODE REVIEW

**Review by ekohl — CHANGES_REQUESTED** [review]
Besides the build failure (which is caused by changes in the `%files` section), you should also package rbvmomi2 because repoclosure will fail on that once you fix the build failure. That can be done in the same PR using `./add_gem_package.sh rbvmomi2 default foreman foreman-el8`.

**Comment by chris1984** [comment]
@ekohl looks like it's failing since rbvmomi2 is not in a list somewhere that obal lint is checking, I will look to see where else it needs added in the meantime till I hear back from you.

**Comment by chris1984** [comment]
Figured it out, had to add it to the package_manifest

**Comment by ekohl** [comment]
Looks like the logs were rotated so [test rpm]

File: packages/foreman/rubygem-fog-vsphere/rubygem-fog-vsphere.spec [hunk 1]
```diff
@@ -2,7 +2,7 @@
 %global gem_name fog-vsphere
 
 Name: rubygem-%{gem_name}
-Version: 3.5.2
+Version: 3.6.0
```
Comment (ekohl) [inline packages/foreman/rubygem-fog-vsphere/rubygem-fog-vsphere.spec:5]: Looking at https://github.com/theforeman/foreman-packaging/pull/8856 you're missing some updates to the minimum Ruby version. Are you using `./bump_rpm.sh` to do updates?

**Review by ekohl — CHANGES_REQUESTED** [review]

**Review by ekohl — CHANGES_REQUESTED** [review]
You also need to reflect the changed files in fog-vsphere:
```
Checking for unpackaged file(s): /usr/lib/rpm/check-files /builddir/build/BUILDROOT/rubygem-fog-vsphere-3.6.0-1.el8.noarch
error: Installed (but unpackaged) file(s) found:
   /usr/share/gems/gems/fog-vsphere-3.6.0/.github/workflows/ruby.yml
    File not found: /builddir/build/BUILDROOT/rubygem-fog-vsphere-3.6.0-1.el8.noarch/usr/share/gems/gems/fog-vsphere-3.6.0/.travis.yml
    Installed (but unpackaged) file(s) found:
   /usr/share/gems/gems/fog-vsphere-3.6.0/.github/workflows/ruby.yml
```

Please squash this in the right commits.

**Comment by chris1984** [comment]
@ekohl Do you know why this is not building? I fixed the rebase like you suggested so that part is fine now.

**Comment by ekohl** [comment]
> You also need to reflect the changed files in fog-vsphere:
> 
> ```
> Checking for unpackaged file(s): /usr/lib/rpm/check-files /builddir/build/BUILDROOT/rubygem-fog-vsphere-3.6.0-1.el8.noarch
> error: Installed (but unpackaged) file(s) found:
>    /usr/share/gems/gems/fog-vsphere-3.6.0/.github/workflows/ruby.yml
>     File not found: /builddir/build/BUILDROOT/rubygem-fog-vsphere-3.6.0-1.el8.noarch/usr/share/gems/gems/fog-vsphere-3.6.0/.travis.yml

To fix that you'll need to drop this:

https://github.com/theforeman/foreman-packaging/blob/09755ad90f84b2c117b153c17f8011b9a555ed78/packages/foreman/rubygem-fog-vsphere/rubygem-fog-vsphere.spec#L53

>     Installed (but unpackaged) file(s) found:
>    /usr/share/gems/gems/fog-vsphere-3.6.0/.github/workflows/ruby.yml
> ```

And to fix this you'll need a a similar `%exclude %{gem_instdir}/.github`. Do mind that you keep it sorted, so insert it around `.gitignore`.

**Comment by ekohl** [comment]
@chris1984 can we move this forward? Or should I submit a replacement PR that resolves it? For now I've held that back, but this really needs to get in.

**Comment by chris1984** [comment]
Will fix it up today

**Review by ekohl — APPROVED** [review]
Looks good. Waiting for CI to complete.

**Pull request merged**